pom.xml

```diff
@@ -13,9 +13,10 @@
     <spark.version>3.3.1</spark.version>
     <scalatest.version>3.2.12</scalatest.version>
     <connector.version>3.2.0</connector.version>
-    <cassandra.version>3.11.13</cassandra.version>
+    <cassandra.version>5.0-alpha1</cassandra.version>
```

Comment:
FYI - There is a [datastax/dse-server:7.0.0-a](https://hub.docker.com/r/datastax/dse-server/tags?page=1&name=7.0.0-a) which I didn't want to use it on here (just yet).